solana/rpc/api.py

```diff
@@ -45,6 +49,45 @@ def get_balance(self, pubkey: Union[PublicKey, str], commitment: Commitment = Ma
         """
         return self._provider.make_request(RPCMethod("getBalance"), str(pubkey), {self._comm_key: commitment})
 
+    def get_account_info(
+        self,
+        pubkey: Union[PublicKey, str],
+        commitment: Commitment = Max,
+        encoding: str = "base64",
+        data_slice: Optional[DataSlice] = None,
+    ) -> RPCResponse:
+        """Returns all the account info for the specified public key.
+
+        :param pubkey: Pubkey of account to query, as base-58 encoded string or PublicKey object.
+        :param commitment: which bank state to query. It can be either "max", "root", "single" or "recent".
+        :param encoding: (optional) encoding for Account data, either "base58" (slow), "base64", or "jsonParsed".
+            Default to "base64".
+            "base58" is limited to Account data of less than 128 bytes.
+            "base64" will return base64 encoded data for Account data of any size.
+            "Parsed-JSON" encoding attempts to use program-specific state parsers to return more human-readable and
+            explicit account state data.
+            If parsed-JSON is requested but a parser cannot be found, the field falls back to base64 encoding,
+            detectable when the data field is type. (jsonParsed encoding is UNSTABLE).
+        :param data_slice: (optional) limit the returned account data using the provided `offset`: <usize> and
+            `length`: <usize> fields; only available for "base58" or "base64" encoding.
+
+        >>> from solana.publickey import PublicKey
+        >>> solana_client = Client("http://localhost:8899")
+        >>> solana_client.get_account_info(PublicKey(1)) # doctest: +SKIP
+        {'jsonrpc': '2.0',
+         'result': {'context': {'slot': 33265073},
+          'value': {'data': '',
+           'executable': False,
+           'lamports': 4459816188034584,
+           'owner': '11111111111111111111111111111111',
+           'rentEpoch': 90}},
+         'id': 1}
+        """
+        opts: Dict[str, str] = {self._encoding_key: encoding, self._comm_key: commitment}
+        if data_slice:
+            opts[self._data_slice_key] = data_slice.__repr__()
```

Comment:
This won't work:

```py
from solana.rpc.types import RPCMethod
solana_client._provider.make_request(RPCMethod("getAccountInfo"), "4Nd1mBQtrMJVYVfKf2PJy9NZUZdTAsp7D4xWLs4gDB4T", 'DataSlice(offset=1, length=1)'
)
# Returns
{'jsonrpc': '2.0',
 'error': {'code': -32602,
  'message': 'Invalid params: invalid type: string "DataSlice(offset=1, length=1)", expected struct RpcAccountInfoConfig.'},
 'id': 2}
```

I think the API expected `{"DataSlice": {"offset": 1, "length": 1"}`.